osquery/sql/virtual_table.cpp

```diff
@@ -658,6 +660,13 @@ int xCreate(sqlite3* db,
     }
   }
 
+  if (is_extension && FLAGS_extensions_default_index &&
+      !extensionSetsOptionColumn) {
+    LOG(INFO) << "Deprecation warning: extension table " << name
```

Comment:
I'm not sure how I feel about this being `LOG(INFO)`. That seems loud. Could this be a VLOG?

Reply:
I'm not sure if deprecations will be useful if only in verbose, which is typically for troubleshooting.  The log line here in xCreate gets called only once per extension table registration.

Reply:
My initial thought, is that until we have a depreciation plan. we should be pretty quiet. 

Unless we're planning on changing the defaults in the near term, I think we can use VLOG

Reply:
I agree, once we have a plan and all extension SDKs (cpp, python, go, etc) support options we can add the warning. If we land this with INFO people are going to ask: 'what action do I take' and we do not have resolutions for everyone.